client/store.js

```diff
@@ -344,7 +344,8 @@ store.setupKresus = function(cb) {
 
             bank.accounts = new Map;
             for (let accPOD of accounts) {
-                let acc = new Account(accPOD);
+                let defaultCurrency = store.getSetting('defaultCurrency');
```

Comment:
Can you put this line just above the `for` loop, please? (to make sure it's not recomputed in every loop turn)